src/domain/collaboration/callout/callout.service.authorization.ts

```diff
@@ -61,6 +63,18 @@ export class CalloutAuthorizationService {
       );
     }
 
+    // TODO: REMOVE ME - Copied from aspects.service.authorization.ts:38, not sure what this code does tbh
+
+    // Inherit for comments before extending so that the creating user does not
+    // have rights to delete comments
+    if (callout.comments) {
+      callout.comments =
+        await this.commentsAuthorizationService.applyAuthorizationPolicy(
+          callout.comments,
+          callout.authorization
+        );
+    }
```

Comment:
This to prevent the creator of the Callout to be able to delete all comments on that Callout. It should stay here.